### DISCUSSION TITLE
Will VS Code Extension Signature Verification Prevent installs in Code Server?

### DISCUSSION THREAD

**Discussion by jonnyhork** [coder/code-server#6037] [General] [answered]
I am trying to anticipate issues with installing extensions in Code Server considering VS Code, starting with version 1.75, will [prevent extensions from being installed that are not signed by the VS Code Marketplace](https://code.visualstudio.com/updates/v1_75#_vs-marketplace-extension-signing), at least from what I can tell. Since Code Server v4.10.0 will be based on VS Code v 1.75.1, I am wondering if there will be any issues installing extensions from OpenVSX?

In the VS Code release notes linked above it talked about rolling out the verification with a feature flag. I am not sure where they are in the process but I do see some PRs related to the feature https://github.com/microsoft/vscode/issues/162284

I am curious if anyone can clarify how this will effect Code Server, if at all?

**Comment by code-asher** [comment, marked as answer]
For now it seems only signed extensions are validated so I believe this will not affect unsigned extensions from Open VSX at least for 1.75.  **Edit**: double-checked just now and am able to install extensions from Open VSX.

In the future if unsigned extensions are also disallowed then we may have to revisit.  There appears to be an `extensions.verifySignature` configuration option that can toggle the behavior.

**Reply by jonnyhork** [reply]
Thanks for checking!
